src/index.ts

```diff
@@ -89,6 +89,12 @@ export type ModelEntry = [string, string, 0 | 1, ModelEntryScreen[]];
 
 const debug = false ? console.log : undefined;
 
+if (!String.prototype.includes) {
```

Comment:
Hi, sorry I haven't had time yet to look at your MR in more detail. I was thinking a bit more about compatibility and I don't think it is a good idea / necessary to include the String .includes polyfill inside of the library.